src/providers/Gardens.js

```diff
@@ -23,30 +23,26 @@ import { getGardenForumUrl } from '../utils/garden-utils'
 import useGardenFilters from '@/hooks/useGardenFilters'
 import { testNameFilter } from '@/utils/garden-filters-utils'
 import { useDebounce } from '@/hooks/useDebounce'
+import { useMounted } from '@/hooks/useMounted'
 
 import { getVoidedGardensByNetwork } from '../voided-gardens'
+import { getGarden } from '@1hive/connect-gardens/dist/cjs/gardens'
 
 const DAOContext = React.createContext()
 
 export function GardensProvider({ children }) {
   const [queryFilters, filters] = useGardenFilters()
-  const [gardens, gardensMetadata, loading, reload] = useGardensList(
+  const [gardens, gardensMetadata, gardensLoading, reload] = useGardensList(
```

Comment:
With the new changes to support multichain. The logic of this file is going to change a bit given that we will use the `preferredNetwork` as chainId instead of `getNetwork().chainId`.